### PR TITLE
Add neumann

### PR DESCRIPTION
**Pull request by imstar15**
OAK Testnet has been abandoned, and we launched a new Neumann network.

Therefore, we need to submit this PR to replace it.

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -315,10 +315,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       }
     },
     {
-      info: 'oak-testnet',
-      text: t('rpc.test.oak', 'OAK Testnet', { ns: 'apps-config' }),
+      info: 'neumann',
+      text: t('rpc.test.oak', 'Neumann Network', { ns: 'apps-config' }),
       providers: {
-        'OAK Network': 'wss://rpc.testnet.oak.tech'
+        'OAK Network': 'wss://neumann.api.onfinality.io/public-ws'
```
Comment (irsal) [inline packages/apps-config/src/endpoints/testing.ts:321]: Replace `OAK Network` with `OnFinality`. We'll do a different `OAK Network` provider at a later time.

File: packages/apps/public/locales/en/apps-config.json [hunk 2]
```diff
@@ -198,7 +198,7 @@
   "rpc.test.mybank": "mybank.network",
   "rpc.test.nftmart": "NFTMart",
   "rpc.test.nodle-arcadia": "Arcadia",
-  "rpc.test.oak": "OAK Testnet",
+  "rpc.test.oak": "Neumann Network",
```
Comment (irsal) [inline packages/apps/public/locales/en/apps-config.json:201]: This is fine.

**Review by irsal — APPROVED** [review]
LGTM (OAK).

**Comment by imstar15** [comment]
Please help review this PR. Thanks!

@jacogr

**Review by jacogr — APPROVED** [review]

**Pull request merged**